packages/node-core/src/meta/meta.service.ts

```diff
@@ -29,8 +29,8 @@ export abstract class BaseMetaService {
   private lastProcessedTimestamp?: number;
   private processedBlockCount?: number;
 
-  constructor(private storeCacheService: StoreCacheService, private config: NodeConfig) {
-    UPDATE_HEIGHT_INTERVAL = config.storeFlushInterval * 1000;
+  constructor(private storeCacheService: StoreCacheService, config: NodeConfig) {
+    // TODO update UPDATE_HEIGHT_INTERVAL should be configureable based on config.storeFlushInterval * 1000 but need to use SchedulerRegistry for that
```

Comment:
I've left this as a TODO so we don't create a breaking change and can make a patch release. 
Im not sure how important it is to be configurable, and it probably makes sense to remove this entirely and have fetch service be responsible for this with its intervals for finalized/latest